tests/DartTest.sh

```diff
@@ -18,9 +18,9 @@ pushd "$(dirname $0)" >/dev/null
 
 command -v pub >/dev/null 2>&1 || { echo >&2 "Dart tests require `pub` but it's not installed.  Aborting."; exit 1; }
 command -v dart >/dev/null 2>&1 || { echo >&2 "Dart tests require dart to be in path but it's not installed.  Aborting."; exit 1; }
-# output required files to the dart folder so that pub will be able to 
+# output required files to the dart folder so that pub will be able to
 # distribute them and more people can more easily run the dart tests
-../flatc --dart -I include_test -o ../dart/test monster_test.fbs 
+../flatc --dart --no-includes -I include_test -o ../dart/test monster_test.fbs
```

Comment:
Why shouldn't the test include the included files? It seems like most of the other languages don't need to specify this flag.

Reply:
I'm not 100% sure on this one to be honest. This functionality was added by @WoodyGuo. I suspect something is broken here, but I'm not sure what or even how to test it. At any rate, this gets tests passing again so we can make sure we don't regress anything else.

Reply:
Sounds good.